docs/ABOUT.md

```diff
@@ -1,4 +1,4 @@
-Go is an open source programming language started by engineers at Google. It's written using a C-style syntax, has statically typed variables, manages memory using garbage collection, and is compiled into stand-alone executables.
+Go is an open source programming language qtarted by engineers at Google. Robert Griesemer, Rob Pike (Unix Programming Environment, Plan 9, UTF-8), and Ken Thompson (of Unix, Plan 9, and UTF-8 fame).  It's written using a C-style syntax, has statically typed variables, manages memory using garbage collection, and is compiled into stand-alone executables.
```

Comment:
we'll want to change "qtarted" back to "started".

I think we should change the period after "Google" to a colon, so that the list of names makes sense. Currently, it is just three names, so it's not a complete sentence.